api.go

```diff
@@ -376,14 +376,15 @@ func (r *Repository) MergeMutable(x *Repository) (mutated bool, err error) {
 // IndexMetadata holds metadata stored in the index file. It contains
 // data generated by the core indexing library.
 type IndexMetadata struct {
-	IndexFormatVersion    int
-	IndexFeatureVersion   int
-	IndexMinReaderVersion int
-	IndexTime             time.Time
-	PlainASCII            bool
-	LanguageMap           map[string]byte
-	ZoektVersion          string
-	ID                    string
+	IndexFormatVersion     int
+	IndexFeatureVersion    int
+	IndexMinReaderVersion  int
+	IndexTime              time.Time
+	PlainASCII             bool
+	LanguageMap            map[string]byte
+	ZoektVersion           string
+	ID                     string
+	DisambiguatedLanguages bool
```

Comment:
Can you add documentation for this field? Is this more of a marker to say go-enry was used to generate LanguageMap? I'm trying to understand why we can't just always use LanguageMap and not care about who generate it?

Reply:
Yes, it indicates that go-enry was used to generate LanguageMap, because ctags-based lang: matching has far fewer language types.